src/pages/governance/index.jsx

```diff
@@ -0,0 +1,88 @@
+import Layout from '@theme/Layout';
+import clsx from 'clsx';
+import React, { useState } from 'react';
+
+import styles from './governance.module.scss';
+
+/**
+ * See "governance" script for _data.json generation
+ */
+import data from './_data.json';
+
+export default function GovernancePage() {
+  return (
+    <Layout title="Governance">
+      <div className="container margin-vert--xl">
```

Comment:
```suggestion
      <main className="container margin-vert--xl">
```

We are missing a main landmark and I believe there is a "hidden link" to jump to the main content in docusaurus as well.